astropy/utils/data_info.py

```diff
@@ -446,6 +446,14 @@ class BaseColumnInfo(DataInfo):
     # depending on context.
     _serialize_context = None
 
+    def __init__(self, bound=False):
+        super().__init__(bound=bound)
```

Comment:
If this is backported it needs the arguments for `super`. Regard this as just a note for the "backporter".

Reply:
@MSeifert04 - Please use the "manual backport" label for these cases, as I'm not reading through the comments when doing backports ;)

Reply:
And so begins the predicted pain of backporting Py3-only code to the LTS...  hopefully it will be manageable.

Reply:
Funnily enough, this was quite painless, and the real conflicts are in pprint.py...

(Though the heads up is really needed as this part obviously wasn't conflicting)